lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -294,6 +294,9 @@ def add_address(post, options)
       def add_creditcard(post, creditcard, options)
         card = {}
         if emv_payment?(creditcard)
+          if creditcard.respond_to?(:contactless) && creditcard.contactless.present?
+            card[:read_method] = "contactless" if creditcard.contactless
```

Comment:
This if seems redundant. The previous line protects this line from executing if there is nothing in contactless.